src/Symfony/Component/CssSelector/Tests/CssSelectorTest.php

```diff
@@ -1,67 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\CssSelector\Tests;
-
-use Symfony\Component\CssSelector\CssSelector;
-
-/**
- * @group legacy
- */
-class CssSelectorTest extends \PHPUnit_Framework_TestCase
-{
-    public function testCssToXPath()
-    {
-        $this->assertEquals('descendant-or-self::*', CssSelector::toXPath(''));
-        $this->assertEquals('descendant-or-self::h1', CssSelector::toXPath('h1'));
-        $this->assertEquals("descendant-or-self::h1[@id = 'foo']", CssSelector::toXPath('h1#foo'));
-        $this->assertEquals("descendant-or-self::h1[@class and contains(concat(' ', normalize-space(@class), ' '), ' foo ')]", CssSelector::toXPath('h1.foo'));
-        $this->assertEquals('descendant-or-self::foo:h1', CssSelector::toXPath('foo|h1'));
-    }
-
-    /** @dataProvider getCssToXPathWithoutPrefixTestData */
-    public function testCssToXPathWithoutPrefix($css, $xpath)
-    {
-        $this->assertEquals($xpath, CssSelector::toXPath($css, ''), '->parse() parses an input string and returns a node');
-    }
-
-    public function testParseExceptions()
-    {
-        try {
-            CssSelector::toXPath('h1:');
-            $this->fail('->parse() throws an Exception if the css selector is not valid');
-        } catch (\Exception $e) {
-            $this->assertInstanceOf('\Symfony\Component\CssSelector\Exception\ParseException', $e, '->parse() throws an Exception if the css selector is not valid');
-            $this->assertEquals('Expected identifier, but <eof at 3> found.', $e->getMessage(), '->parse() throws an Exception if the css selector is not valid');
-        }
-    }
-
-    public function getCssToXPathWithoutPrefixTestData()
```

Comment:
I'm not sure all these tests where copied to the tests of the new class. It should be checked before removing them